src/main/resources/com/google/api/codegen/ruby/smoke_test.snip

```diff
@@ -23,7 +23,15 @@ describe "{@smokeTest.name}" do
       project_id = ENV["SMOKE_TEST_PROJECT"].freeze
     @end
 
-    {@sampleCodeCore(smokeTest.apiMethod)}
+    process_response = ->(response) {
```

Comment:
Good catch about LRO. Having the gRPC mock return a done operation when the method is LRO would prevent it from blocking.

In any case, because `operation.wait_until_done!` etc. aren't generated, I'm fine with leaving them out of the generated test if that makes it easier.

But that's not the issue though, rather: methods whose response is async can't be verified by appending lines to the sample code. Because the response is within a callback. E.g. for this method:

```ruby
library_service_client = LibraryServiceClient.new

# Iterate over all results.
library_service_client.list_shelves.each do |element|
  # Process element.
end

# Or iterate over results one page at a time.
library_service_client.list_shelves.each_page do |page|
  # Process each page at a time.
  page.each do |element|
    # Process element.
  end
end
```

We can't just add the verification at the end.